### ISSUE TITLE
The Radar section is not yet fully responsive

### ISSUE THREAD

**Issue by openp2pdesign** [Make-IT-TR/TechRadar#15]
<img width="913" alt="screen shot 2017-06-19 at 12 15 07" src="https://user-images.githubusercontent.com/1458229/27280676-14ac7a98-54e9-11e7-95fa-f600e2b88f91.png">

The Radar section is not fully responsive yet and it gets cut on the right side, this would make it better on mobiles and when embedded.

**Comment by openp2pdesign** [comment]
One of the reasons why the radar is not responsive is that the technologies are added as [a second svg with absolute position on top of the svg of the radar](https://github.com/Make-IT-TR/TechRadar/blob/master/tr-app/src/radar/radar.ts#L221). The common procedure with D3 would be to add them to a group inside the same svg, this creates several issues...

**Comment by openp2pdesign** [comment]
For example, mouseover on the radar does not work since there's another div/svg on top of it (note that the mouseover has other issues... 

```
radar.ts:395 Uncaught TypeError: Cannot read property 'appendChild' of undefined
    at SVGPathElement.<anonymous> (radar.ts:395)
    at SVGPathElement.__onmouseover (d3.js:1120)
```

**Comment by openp2pdesign** [comment]
This is now fixed with PR #18 , once @damylen merges it. I changed the layout to make it both more understandable and more responsive (also the D3 code is now responsive, not just the html)

![screen shot 2017-08-07 at 16 53 50](https://user-images.githubusercontent.com/1458229/29032404-90e6803a-7b91-11e7-892d-76698c0d728d.png)

**Comment by openp2pdesign** [comment]
Fixed with #18
